utils/database_utils/mongo_utils/structures.py

```diff
@@ -52,6 +52,8 @@ class MongoCommands(Enum):
     DELETE_MANY = "delete_many"
     # Update operation
     UPDATE = "update"
```

Comment:
is `update` valid still? In https://github.com/NeonGeckoCom/pyklatchat/pull/71 I replaced this with `update_many` because `update` calls were failing